test/libyul/yulSyntaxTests/eof/call_intruction_in_eof.yul

```diff
@@ -0,0 +1,13 @@
+object "a" {
+    code {
+        let success := call(gas(), 0x1, 0, 128, 4, 128, 0)
+    }
+}
+// ====
+// EVMVersion: >=prague
```

Comment:
That's not overly important to change, but now that I think of it, instead of the extra test we could also have just done this:

```suggestion
// EVMVersion: >=shanghai
```

This would be enough to make sure that the path with `EVMVersion` predating EOF in `EVMVersionRestrictedTestCase::processBytecodeFormatSetting()` is covered by at least one test.

Reply:
Changed.